status-im/status-react
# Enable custom mailservers in nightly/releases

fixes #4299
fixes #4328
fixes #4586

### Summary:

This commit enable custom mailservers for nightly & release.

### Review notes:

Only config change, but this will go through manual QA.

### Testing notes:

You should be able to add and use custom mailservers:

1) Go to profile
2) Advance
3) Custom Mailservers
4) Click on '+'
5) Enter name & address(without quotes): "enode://77a3a6e179ceac552cb4689bee3ea5fca4003e77a5b470540c27fcfaba18333a43d3b89f3b7f12e386acf8bb340d39da74b5288e44f54cedf24c20f678cade7d:testtesttest@167.99.219.240:30504"
6) Click again on the mailserver just created
7) Click on Connect
8) You should be logged out
9) Join `#status` 
10) You should receive some messages from offline inbox ( this mailserver is a test mailserver, so not sure how many / if they are the same as the normal one, just the fact you received some is enough to tell it works).

- You should be able to edit custom mailservers by clicking on it/editing/ click save. 
- You should be able to delete custom mailservers by clicking on it & press Delete
- You should be able to scan a qr code to add a mailserver.

If the mailserver is in use you can only edit, no delete, and you will be logged out on successful save.

### Outstanding issues

If the url/password is wrong no indication is given ( apart from the fact that messages are not received). This is due to the fact that the underlying request report success, so for now won't be addressed.

Depends on: #4517 

status: ready
